prac_06/programming_language.py

```diff
@@ -0,0 +1,18 @@
+class ProgrammingLanguage:
+    """New class for categorising programming languages."""
+
+    def __init__(self, name="", typing="", reflection=False, year=""):
```

Comment:
setting name, typing and year ="" and setting reflection to False is not needed. keeping them there shouldn't cause any harm but using a line like:
def __init__(self, name, typing, reflection, year):
would also work as you are then later setting what each of them are right after inside the same function.